example-streamer-uses/src/bin/common/mod.rs

```diff
@@ -0,0 +1,94 @@
+use async_trait::async_trait;
+use hello_world_protos::{
+    hello_world_service::{HelloRequest, HelloResponse},
+    hello_world_topics::Timer,
+};
+use log::{debug, error, info};
+use protobuf::Message;
+use std::sync::Arc;
+use up_rust::{UListener, UMessage, UMessageBuilder, UTransport};
+
+#[allow(dead_code)]
```

Comment:
With the factoring out of the common structs now clippy complains because whenever you run a service, the response listener is unused and whenever you run a client, the request listener is unused. Could also solve this with some kind of matching function (or maybe even more elegant but Im not seeing it) but since its only in the example code I feel like this way is cleaner.